patternfly/patternfly-elements
# Fix | band layout

## Title of this pull request

* Link to discovery documentation (should contain all the planning and requirements for this work):
   - See the `DISCOVERY.md` template at the root of the project
* Link(s) to demo pages where this element can be viewed: http://localhost:8000/elements/pfe-band/demo/
* [Github Issue](https://github.com/patternfly/patternfly-elements/issues/330)

---

### What has changed and why
* Band layout was broken on IE, Edge and FF 60 and below. Solution was adding grid-areas in the lightdom for edge and for IE adding float. 

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. Run [demo page](http://localhost:8000/elements/pfe-band/demo/) through browser testing to verify bug is fixed

#### Browser requirements
Your component should work in all of the following environments:


Your repository infrastructure updates should work for at least:

### Ready-for-merge Checklist



**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
